content/tutorial/01-svelte/01-introduction/06-html-tags/README.md

effy-coding opened the pull request.

effy-coding committed:
content/tutorial/01-svelte/01-introduction/06-html-tags/README.md

Jueun-Park commented on content/tutorial/01-svelte/01-introduction/06-html-tags/README.md:
```diff
@@ -2,15 +2,15 @@
 ---
 
-Ordinarily, strings are inserted as plain text, meaning that characters like `<` and `>` have no special meaning.
+일반적으로 문자열은 일반 텍스트로 삽입되므로 < 및 >과 같은 문자는 특별한 의미가 없습니다.
```
`<`, `>` 문자를 코드블록으로 표시해야 합니다.

effy-coding replied:
넵 제가 이건 놓쳤네요 수정해서 반영했습니다

Jueun-Park commented on content/tutorial/01-svelte/01-introduction/06-html-tags/README.md:
```diff
@@ -2,15 +2,15 @@
+하지만 때로는 HTML을 컴포넌트에 직접 렌더링해야 할 때가 있습니다. 예를 들어, 지금 읽고 있는 단어는 마크다운 파일에 존재하며 이 페이지에 HTML 덩어리로 포함됩니다.
 
-In Svelte, you do this with the special `{@html ...}` tag:
+Svelte에서는 특수 {@html ...} 태그를 사용하여 이 작업을 수행합니다:
```
`{@html ...}` 문자를 코드블록으로 표시해야 합니다.

effy-coding replied:
여기도 동일하게 수정 및 반영했습니다

Jueun-Park commented on content/tutorial/01-svelte/01-introduction/06-html-tags/README.md:
````diff
@@ -2,15 +2,15 @@
 ```
 
-> **Warning!** Svelte doesn't perform any sanitization of the expression inside `{@html ...}` before it gets inserted into the DOM. This isn't an issue if the content is something you trust like an article you wrote yourself. However if it's some untrusted user content, e.g. a comment on an article, then it's critical that you manually escape it, otherwise you risk exposing your users to <a href= ...
+> **경고!** Svelte는 {@html ...} 내부의 표현식이 DOM에 삽입되기 전에 어떠한 sanitization도 수행하지 않습니다. 콘텐츠가 직접 작성한 글처럼 신뢰할 수 있는 내용이라면 문제가 되지 않습니다. 그러나 신뢰할 수 없는 사용자 콘텐츠(예: 기사의 댓글)인 경우에는 수동으로 이스케이프 처리해야 하며, 그렇지 않으면 사용자가 <a href="https://owasp.org/www-community/attacks/xss/" target="_blank">크로스 사이트 스크립팅</a> (XSS) 공격에 노출될 위험이 있습니다.
````
- `sanitization`이 번역되지 않았습니다. 어떻게 변역하면 좋을지 의견이 필요합니다.
  - `어떠한 편집도 수행하지 않습니다.`
  - `어떠한 수정과 삭제도 수행하지 않습니다.`

다른 의견도 모두 환영합니다!

---
[참고] `Cross-Site Scripting`과 같은 외국어를 어떻게 표현해야 할 지에 대한 컨벤션이 아직 논의 중입니다. (#8 )
- 크로스-사이트 스크립팅(Cross-Site Scripting, XSS)
- 크로스-사이트 스크립팅(XSS)
- Cross-Site Scripting(XSS)

D0ri123 replied:
`어떤 변경이나 삭제도 이뤄지지 않습니다.` 는 어떠신가요오~??

---
크로스-사이트 스크립팅(Cross-Site Scripting, XSS)으로 표기하는게 좋다고 생각합니다! 
다만 우려되는 건 '한글 + 영어 + 축약어' 를 매번 본문에 계속 언급하면 오히려 가독성을 떨어지게 만드는 거 같아서요.

처음 표기할 때 => 크로스-사이트 스크립팅(Cross-Site Scripting, XSS)
두번째부터는 => Cross-Site Scripting

이런 방식은 어떨까요?

effy-coding replied:
`sanitization` 는 한글로 어떻게 표기해야 할지 애매해서 미번역 했었는데 후자로 변경해봤어요. 편집은 사람이 하는거라면 수정과 삭제는 좀 더 프로그래마틱한 뉘앙스를 풍겨서요

추가로 @D0ri123 님이 의견주신대로 XSS쪽도 수정해봤습니다.

Jueun-Park reviewed (changes requested):
의견 확인 부탁드립니다 ㅎㅎ

effy-coding committed:
Update README.md

<와 >를 코드 블럭으로 함
{@html ...}를 코드 블럭으로 함
sanitization을 번역함
크로스 사이트 스크립팅를 크로스-사이트 스크립팅(Cross-Site Scripting, XSS)로 바꿔봄

Jueun-Park reviewed (approved):
lgtm
